src/lib.rs

```diff
@@ -219,12 +239,14 @@ mod tests {
         }
 
         impl super::OutputPin for &'_ mut TestPin {
-            fn set_high(&mut self) {
-                (*self).0 = 1;
+            type Error = ();
+
+            fn set_high(&mut self) -> core::result::Result<(), Self::Error> {
+                Ok((*self).0 = 1)
             }
 
-            fn set_low(&mut self) {
-                (*self).0 = 0;
+            fn set_low(&mut self) -> core::result::Result<(), Self::Error> {
+                Ok((*self).0 = 0)
```

Comment:
While strictly valid, I find code like this a bit confusing. I'd prefer this:

```rust
(*self).0 = 0;
Ok(())
```

Reply:
Ok!